cca_realm_claims_test.go

```diff
@@ -0,0 +1,273 @@
+package ccatoken
+
+import (
+	"os"
+	"testing"
+
+	"github.com/stretchr/testify/assert"
+	"github.com/stretchr/testify/require"
+)
+
+func mustBuildValidCcaRealmClaims(t *testing.T) IClaims {
+	c, err := NewClaims()
+	require.NoError(t, err)
+
+	err = c.SetChallenge(testChallenge)
+	require.NoError(t, err)
+
+	err = c.SetPersonalizationValue(testPersonalizationVal)
+	require.NoError(t, err)
+
+	err = c.SetInitialMeasurements(testInitMeas)
+	require.NoError(t, err)
+
+	err = c.SetExtensibleMeasurements(testExtensibleMeas)
+	require.NoError(t, err)
+
+	err = c.SetHashAlgID(testHashAlgID)
+	require.NoError(t, err)
+
+	err = c.SetPubKey(testPubKey)
+	require.NoError(t, err)
+
+	err = c.SetPubKeyHashAlgID(testPubKeyHashAlgID)
+	require.NoError(t, err)
+
+	return c
+}
+
+func Test_NewCca_Realm_Claims_ok(t *testing.T) {
```

Comment:
typo: rather than Cca_Realm_Claims we usually have the name of the object (which is CcaRealmClaims